spec/duration.html

```diff
@@ -591,12 +591,12 @@ <h1>RegulateDuration ( _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _
     </emu-clause>
 
     <emu-clause id="sec-temporal-addduration" aoid="AddDuration">
-      <h1>AddDuration ( _y1_, _mon1_, _w1_, _d1_, _h1_, _min1_, _s1_, _ms1_, _µs1_, _ns1_, _y2_, _mon2_, _w2_, _d2_, _h2_, _min2_, _s2_, _ms2_, _µs2_, _ns2_, _disambiguation_ )</h1>
+      <h1>AddDuration ( _y1_, _mon1_, _w1_, _d1_, _h1_, _min1_, _s1_, _ms1_, _mus1_, _ns1_, _y2_, _mon2_, _w2_, _d2_, _h2_, _min2_, _s2_, _ms2_, _mus2_, _ns2_, _disambiguation_ )</h1>
```

Comment:
I had a feeling when I saw this that it wasn't a good idea. Nit: `mis` vs `mus`?

Reply:
I still think it ought to be fine to have non-ASCII identifiers in the spec but I figured I was not about to spend the time to update the `[A-Za-z][A-Za-z0-9]*` regex to allow that :stuck_out_tongue_closed_eyes:
`mis` could get confused with milliseconds, so maybe `mcs` or just write out the full names after all?

Reply:
True. It's a nitpick anyway, I have no real preference.